packages/react/package.json

```diff
@@ -78,7 +78,6 @@
   "dependencies": {
     "@clerk/shared": "2.0.0-beta-v5.11",
     "@clerk/types": "4.0.0-beta-v5.13",
-    "eslint-config-custom": "*",
```

Comment:
@dimkl why are we removing this one ?

Reply:
Oh, I should have taken a look at the whole file 😞

Reply:
I will remove it in a separate PR since i am having some issues